src/orquestra/sdk/_base/_environment.py

```diff
@@ -0,0 +1,5 @@
+CONFIG_PATH_ENV = "ORQ_CONFIG_PATH"
```

Comment:
**question**: did you consider using this module to document the env variable purpose and accepted values?

Reply:
done 33071f7